requirements.txt

```diff
@@ -1,2 +1,3 @@
 apypie>=0.0.5
```

Comment:
Btw. Don't we need 0.1.0 here?

Reply:
not strictly, I don't think we *depend* on any 0.1 features (but then again, I didn't test it either)